lib/metrics/eventLoopLag.js

```diff
@@ -1,5 +1,22 @@
 'use strict';
 
+// Notes:
+//
+// Because this collected async, the value will always lag by one scrape
+// interval.
+//
+// Reconcile with:
+// https://nodejs.org/api/perf_hooks.html#perf_hooks_perf_hooks_monitoreventloopdelay_options
+//
+// (perf_hooks exists only `>= v11.10.0`)
+//
+// - Delay would be a prom histogram, but perf_hooks doesn't support that yet.
+// - Current possibility is to report h.max as the gauge value, which is
+// somewhat compatible with the current code.
+// - Might be possible to report perf_hook's histogram as a summary, using its
+// percentile() function. Summaries are less than ideal with prometheus but at
+// least the data is available in 12.x+. See #309
```

Comment:
```suggestion
// least the data is available in 12.x+. See https://github.com/siimon/prom-client/issues/309
```

Just easier for people if they come over this in `node_modules` and want to take a look.

👏 for detailed comment, btw!